docs/dev/table/sql.md

```diff
@@ -1842,6 +1842,16 @@ RPAD(text string, len integer, pad string)
     <tr>
       <td>
         {% highlight text %}
+REGEX_EXTRACT(str string, regex string, extractIndex integer)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns the string str extracted using specified regex pattern and index. If str or regex is null, returns null. E.g. <code>REGEX_EXTRACT('foothebar', 'foo(.*?)(bar)', 2)</code> returns <code>bar</code>.</p>
```

Comment:
This might be more informative:
```
Return the string extracted from the `extractedIndex` capturing group using specified `regex` pattern on input string `str`.
```